Zend/zend_builtin_functions.c

```diff
@@ -917,25 +917,14 @@ static int same_name(zend_string *key, zend_string *name) /* {{{ */
    Returns an array of method names for class or class instance. */
 ZEND_FUNCTION(get_class_methods)
 {
-	zval *klass;
 	zval method_name;
 	zend_class_entry *ce = NULL;
 	zend_class_entry *scope;
 	zend_function *mptr;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "z", &klass) == FAILURE) {
-		RETURN_THROWS();
-	}
-
-	if (Z_TYPE_P(klass) == IS_OBJECT) {
-		ce = Z_OBJCE_P(klass);
-	} else if (Z_TYPE_P(klass) == IS_STRING) {
-		ce = zend_lookup_class(Z_STR_P(klass));
-	}
-
-	if (!ce) {
-		RETURN_NULL();
-	}
+	ZEND_PARSE_PARAMETERS_START_EX(ZEND_PARSE_PARAMS_QUIET, 1, 1)
```

Comment:
We shouldn't use PARSE_PARAMS_QUIET here. This will throw an exception for invalid classes, but that's okay for PHP 8.